Content.Shared/_Goobstation/MartialArts/SharedMartialArtsSystem.CQC.cs

```diff
@@ -118,25 +118,29 @@ private void OnCQCSlam(Entity<CanPerformComboComponent> ent, ref CqcSlamPerforme
     private void OnCQCKick(Entity<CanPerformComboComponent> ent, ref CqcKickPerformedEvent args)
     {
         if (!_proto.TryIndex(ent.Comp.BeingPerformed, out var proto)
-            || !TryUseMartialArt(ent, proto.MartialArtsForm, out var target, out var downed)
-            || !downed)
+            || !TryUseMartialArt(ent, proto.MartialArtsForm, out var target, out var downed))
             return;
 
-        if (TryComp<StaminaComponent>(target, out var stamina) && stamina.Critical)
-        {
-            _status.TryAddStatusEffect<ForcedSleepingComponent>(target, "ForcedSleep", TimeSpan.FromSeconds(10), true);
-        }
-
-        DoDamage(ent, target, proto.DamageType, proto.ExtraDamage, out var damage, TargetBodyPart.Head);
-        _stamina.TakeStaminaDamage(target, proto.StaminaDamage, source: ent);
-
         var mapPos = _transform.GetMapCoordinates(ent).Position;
         var hitPos = _transform.GetMapCoordinates(target).Position;
         var dir = hitPos - mapPos;
         dir *= 1f / dir.Length();
+
+        if (downed)
+        {
+            if (TryComp<StaminaComponent>(target, out var stamina) && stamina.Critical)
+                _status.TryAddStatusEffect<ForcedSleepingComponent>(target, "ForcedSleep", TimeSpan.FromSeconds(10), true);
+            DoDamage(ent, target, proto.DamageType, proto.ExtraDamage, out _, TargetBodyPart.Head);
```

Comment:
```suggestion
                _status.TryAddStatusEffect<ForcedSleepingComponent>(target, "ForcedSleep", TimeSpan.FromSeconds(10), true);
                
            DoDamage(ent, target, proto.DamageType, proto.ExtraDamage, out _, TargetBodyPart.Head);
```
I like some blank line for readability

Reply:
Fair